pkg/controller/petset/pet.go

```diff
@@ -216,6 +220,7 @@ func (p *apiServerPetClient) Update(pet *pcb, expectedPet *pcb) (updateErr error
 		if p, getErr = pc.Get(pod.Name); getErr != nil {
 			return getErr
 		}
+		time.Sleep(retrySleepTime)
```

Comment:
Well, I think I have realized the delay queue you mentioned before. But as far as I know, we enqueue the petset only if `Update()` return an error here. Actually, `Update` failure won't return an error if it doesn't exceed the `updateRetries` count.

However, this PR is about other issues - whether we need to sleep between update(pod) failures?